workers/unity/Assets/Gdk/Core/Error/ILogDispatcher.cs

```diff
@@ -0,0 +1,9 @@
+using UnityEngine;
+
+namespace Improbable.Gdk.Core
+{
+    public interface ILogDispatcher
+    {
+        void HandleLog(LogEvent logEvent);
```

Comment:
Does it make more sense to detach the log level from the LogEvent, e.g. changing the signature to `HandleLog(LogType, LogEvent)`?
That also opens up a more Unity-like API of `LogError`, `LogWarning`, etc.

In fact, Unity already has `ILogHandler` - https://docs.unity3d.com/ScriptReference/ILogHandler.html
Was this taken into account during the design phase?

```
interface ILogHandler
    {
        public void LogFormat(LogType logType, Object context, string format, params object[] args);
        public void LogException(Exception exception, Object context);
    }
```

Reply:
Changed the signature as suggested above :)